server/data/db.go

```diff
@@ -35,3 +36,12 @@ func Connect() {
 func Close() {
 	db.Close()
 }
+
+// Function to set default env values, if they don't exist
+// interface{} is used, because we need to return both strings and ints
+func getEnv(key string, fallback interface{}) interface{} {
```

Comment:
I love that you made a helper function for this. The only issue would be the return type.

According to the [docs](https://pkg.go.dev/os#Getenv), `os.Getenv` only returns a `string` and never an `int`.
So maybe using `interface{}` is no needed? After all, using `interface{}` makes you lose all type information (think pure JavaScript lol).